xml/ha_maintenance.xml

```diff
@@ -0,0 +1,548 @@
+    before shutting down or rebooting the node, proceed as follows:
+   </para>
+   <step>
+    <para>
+     On the node you want to reboot or shut down, log in as &rootuser; or
+     equivalent.
+    </para>
+   </step>
+   <step>
+    <para>
+     Put the node into <literal>standby</literal> mode:
+    </para>
+<screen>&prompt.root;crm node standby</screen>
+    <para>
+     That way, services can migrate off the node without being limited by the
+     shutdown timeout of &pace;.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check the cluster status with:
+    </para>
+<screen>&prompt.root;crm status</screen>
+    <para>
+     It shows the respective node in <literal>standby</literal> mode:
+    </para>
+<screen>[...]
+Node <replaceable>&node2;</replaceable>: standby
+[...]</screen>
+   </step>
+   <step>
+    <para>
+     If you have a DLM resource (or other resources depending on DLM), make
+     sure to explicitly stop those resources before stopping the &pace;
+     service:
+    </para>
+<screen>&prompt.crm.res;stop <replaceable>RESOURCE_ID</replaceable></screen>
+    <para>
+     The reason is that stopping &pace; also stops the &corosync; service, on
+     whose membership and messaging services DLM depends. If &corosync; stops,
+     the DLM resource will assume a split-brain scenario and trigger a fencing
+     operation.
+    </para>
+   </step>
+   <step>
+    <para>
+     Stop the &pace; service on that node:
+    </para>
+<screen>&prompt.root;systemctl stop pacemaker.service</screen>
+   </step>
+   <step>
+    <para>
+     Reboot the node.
+    </para>
+   </step>
+  </procedure>
+
+  <para>
+   To check if the node joins the cluster again:
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Log in to the node as &rootuser; or equivalent.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check if the &pace; service has started:
+    </para>
+<screen>&prompt.root;systemctl status pacemaker.service</screen>
+   </step>
+   <step>
+    <para>
+     If not, start it:
+    </para>
+<screen>&prompt.root;systemctl start pacemaker.service</screen>
+   </step>
+   <step>
+    <para>
+     Check the cluster status with:
+    </para>
+<screen>&prompt.root;crm status</screen>
+    <para>
+     It should show the node coming online again.
+    </para>
+   </step>
+  </procedure>
+ </sect1>
+ <sect1 xml:id="sec.ha.maint.overview">
+  <title>Different Options for Maintenance Tasks</title>
+
+  <para>
+   &pace; offers a variety of options for performing system maintenance:
+  </para>
+
+  <variablelist>
+   <varlistentry xml:id="vle.ha.maint.mode.cluster">
+    <!--<term>Putting the Cluster Into Maintenance Mode</term>-->
+    <term><xref linkend="sec.ha.maint.mode.cluster" xrefstyle="select:title"/></term>
+    <listitem>
+     <para>
+      The global cluster property <literal>maintenance-mode</literal> allows
+      you to put all resources into unmanaged state at once. The cluster will
```

Comment:
To avoid getting confused with the following "unamanaged mode", how about we say "put all resources into maintenance state" here?